example/port_power does not compile on YARP devel

Error: 
~~~
[ 41%] Building CXX object CMakeFiles/ex0402_port_callback_reply.dir/ex0402_port_callback_reply.cpp.o
/usr/local/opt/ccache/libexec/c++   -I/Users/traversaro/src/codyco-superbuild/libraries/YARP/example/port_power -isystem /Users/traversaro/src/codyco-superbuild/build/install/include  -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.12.sdk   -std=gnu++11 -o CMakeFiles/ex0402_port_callback_reply.dir/ex0402_port_callback_reply.cpp.o -c /Users/traversaro/src/codyco-superbuild/libraries/YARP/example/port_power/ex0402_port_callback_reply.cpp
/Users/traversaro/src/codyco-superbuild/libraries/YARP/example/port_power/ex0402_port_callback_reply.cpp:18:13: error: no matching member function for call to 'add'
        out.add("acknowledge");
        ~~~~^~~
/Users/traversaro/src/codyco-superbuild/build/install/include/yarp/os/Bottle.h:153:10: note: candidate function not viable: no known conversion from 'const char [12]' to 'const yarp::os::Value' for 1st argument
    void add(const Value& value);
         ^
/Users/traversaro/src/codyco-superbuild/build/install/include/yarp/os/Bottle.h:164:10: note: candidate function not viable: no known conversion from 'const char [12]' to 'yarp::os::Value *' for 1st argument
    void add(Value* value);
         ^
1 error generated.
~~~

--- a/example/port_power/ex0001_sender.cpp
+++ b/example/port_power/ex0001_sender.cpp
@@ -17,9 +17,9 @@ int main() {
     while (true) {
         Bottle b;        // Make a place to store things.
         b.clear();
-        b.add("hello");
-        b.add("world");
-        b.add(ct);
+        b.addString("hello");
+        b.addString("world");
+        b.addInt(ct);
         ct++;
         p.write(b);      // Send the data.
         printf("Sent %s\n", b.toString().c_str());

--- a/example/port_power/ex0101_sender.cpp
+++ b/example/port_power/ex0101_sender.cpp
@@ -17,9 +17,9 @@ int main() {
     while (true) {
         Bottle& b = p.prepare(); // Get a place to store things.
         b.clear();  // clear is important - b might be a reused object
-        b.add("hello");
-        b.add("world");
-        b.add(ct);
+        b.addString("hello");
+        b.addString("world");
+        b.addInt(ct);
         ct++;
         printf("Sending %s\n", b.toString().c_str());
         p.write();            // Send the data.

--- a/example/port_power/ex0400_expect_reply.cpp
+++ b/example/port_power/ex0400_expect_reply.cpp
@@ -18,9 +18,9 @@ int main() {
         Bottle in,out;   // Make places to store things.
         // prepare command "out".
         out.clear();
-        out.add("hello");
-        out.add("world");
-        out.add(ct);
+        out.addString("hello");
+        out.addString("world");
+        out.addInt(ct);
         ct++;
         p.write(out,in); // send command, wait for reply.
         // process response "in".

--- a/example/port_power/ex0401_give_reply.cpp
+++ b/example/port_power/ex0401_give_reply.cpp
@@ -18,7 +18,7 @@ int main() {
         p.read(in,true); // Read from the port, warn that we'll be replying.
         printf("Got %s\n", in.toString().c_str());                
         out.clear();
-        out.add("acknowledge");
+        out.addString("acknowledge");
         out.append(in);
         p.reply(out);    // send reply.
     }

--- a/example/port_power/ex0402_port_callback_reply.cpp
+++ b/example/port_power/ex0402_port_callback_reply.cpp
@@ -15,7 +15,7 @@ class DataProcessor : public PortReader {
         // process data "in", prepare "out"
         printf("Got %s\n", in.toString().c_str());  
         out.clear();
-        out.add("acknowledge");
+        out.addString("acknowledge");
         out.append(in);
         ConnectionWriter *returnToSender = connection.getWriter();
         if (returnToSender!=NULL) {

--- a/example/port_power/ex0403_bufferedport_callback_reply.cpp
+++ b/example/port_power/ex0403_bufferedport_callback_reply.cpp
@@ -20,7 +20,7 @@ class DataProcessor : public TypedReaderCallback<Bottle>, public PortReader {
         // process data "in", prepare "out"
         printf("Got message to reply to: %s\n", in.toString().c_str());  
         out.clear();
-        out.add("acknowledge");
+        out.addString("acknowledge");
         out.append(in);
         ConnectionWriter *returnToSender = connection.getWriter();
         if (returnToSender!=NULL) {

